docs/tutorial/index.org

```diff
@@ -1277,6 +1277,26 @@ elisp code.
 <<lst:remove-line-ref-src-block>>
 #+END_SRC
 
+** Update with a new version of EnOS                               :noexport:
+To update this when a new version of EnOS appears change in the
+following the values of ~new-os-version~ and ~new-enos-install~ with
+the value of the next release. Then executes the code with ~C-c C-c~.
+
+#+BEGIN_SRC elisp :results none
```

Comment:
great :)

Reply:
Yes, this should avoid errors while updating the TP.